src/universal_transfer_operator/data_providers/filesystem/google/cloud/gcs.py

```diff
@@ -24,6 +24,8 @@ class GCSDataProvider(BaseFilesystemProviders):
     DataProviders interactions with GS Dataset.
     """
 
+    location_type = FileLocation.GS
```

Comment:
```suggestion
    location_type = FileLocation.GCS
```

Reply:
@phanikumv This is aligned with the URL that is constructed - gs://bucket is correct and gcs://bucket will fail, I think it's good to be aligned. WDYT?

Reply:
Ok its fine then